eva_submission/biosamples_submission.py

```diff
@@ -364,6 +364,13 @@ def __init__(self, metadata_spreadsheet):
         self.reader = EvaXlsxReader(self.metadata_spreadsheet)
         self.sample_data = self.map_metadata_to_bsd_data()
 
+    @staticmethod
+    def serialize(value):
+        """Create a text representation of the value provided"""
+        if isinstance(value, date):
+            return value.strftime('%Y-%m-%d')
```

Comment:
This may be a question for another time but does the metadata include timezone information?

Reply:
If it contains tiimezone info, then it would be better to use the standard UTC format and the following would be required:
return value.strftime('%Y-%m-%d'T%H:%M:%SZ')

Reply:
We're getting a non timezone datetime from Excel that we only need to convert to date for BioSample API consumption.